File: backend/semgrep_rules/tests/python/kisa_sw_25_insecure_random.py
import random
import secrets
def unsafe():
    # ruleid: kisa.sw25.python.insecure-random-for-security
    return random.randint(100000, 999999)
def safe():
    # ok: kisa.sw25.python.insecure-random-for-security
    return secrets.randbelow(900000) + 100000

File: backend/semgrep_rules/tests/javascript/kisa_sw_25_insecure_random.js
const crypto = require("crypto");
function unsafe() {
  // ruleid: kisa.sw25.javascript.insecure-random-for-security
  return Math.random();
}
function safe() {
  // ok: kisa.sw25.javascript.insecure-random-for-security
  return crypto.randomInt(0, 1000000);
}
